frontend/src/components/Mining.vue

```diff
@@ -55,14 +55,17 @@
       </p>
       <p class="immatureBalance" v-if="balanceImmature != '0.00000000'">
         (
-        <span style="opacity: 1">{{balanceImmature}} VTC</span>
+        <span style="opacity: 1">{{balanceImmature}} {{activePayout}}</span>
```

Comment:
nit: i don't think we can have immature balance in other currencies, given that the pool pays us (we don't receive the foreign coin's block reward)

Reply:
Fixed.